src/components/MemberProfileEdit.tsx

```diff
@@ -34,12 +34,13 @@ const MemberProfileEdit: React.FC<MemberProfileEditProps> = ({
   };
```

Comment:
It won't let me comment above this but you want to be careful with line 29 `setProfile({ ...profile, [name]: value });` You can get some race conditions. I suggest changing it to:

`setProfile((prev) => { ...prev, [name]: value });`

Reply:
fixed